.github/workflows/check-media-attachments.yml

```
@@ -0,0 +1,161 @@
+name: Check PR for media attachments when HTML files change
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)